lib/twilio/audiohelper.ts

```diff
@@ -717,15 +749,29 @@ class AudioHelper extends EventEmitter {
   }
 
   /**
-   * Route input stream to the processor if it exists
+   * Route local stream to the processor if it exists.
+   */
+  private _maybeCreateLocalProcessedStream(stream: MediaStream): Promise<MediaStream> {
```

Comment:
This is private but we're accessing it from peerconnection. Let's make it internal.